readme.md

```diff
@@ -1,7 +1,7 @@
 # HTML form
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_html-form/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_html-form/report/html_report/)
+- [DEMO LINK](https://vksazonov.github.io/layout_html-form/)
+- [TEST REPORT LINK](https://vksazonov.github.io/layout_html-form/report/html_report/)
```

Comment:
Don't forget to add Demo and Test report links here
<img width="1010" alt="Screenshot 2023-02-21 at 09 44 56" src="https://user-images.githubusercontent.com/104769168/220280415-02961491-c812-4964-a479-6b5c76cba298.png">